pyproject.toml

```diff
@@ -39,8 +39,15 @@ Source = "https://github.com/py-pdf/pypdf"
 Changelog = "https://pypdf.readthedocs.io/en/latest/meta/CHANGELOG.html"
 
 [project.optional-dependencies]
-full = ["PyCryptodome", "Pillow>=8.0.0"]
-crypto = ["PyCryptodome"]
+full = [
+    "cryptography; python_version >= '3.7'",
+    "PyCryptodome; python_version == '3.6'",
```

Comment:
I wouldn't add PyCryptodome here, as `cryptography` already gives the user the full functionality of pypdf.

Reply:
ah, my bad